qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -43,7 +43,15 @@
 
 
 class ManagedJobSet:
-    """A set of managed jobs."""
+    """A set of managed jobs.
+
+    An instance of this class is returned when you submit experiments using
+    :meth:`IBMQJobManager.run()<qiskit.providers.ibmq.managed.IBMQJobManager.run()>`.
```

Comment:
This could be condensed to:  
:meth:\`IBMQJobManager.run()<IBMQJobManager.run()>\`